src/parser/mod.rs

```diff
@@ -13595,10 +13619,33 @@ impl<'a> Parser<'a> {
         }
     }
 
-    /// Parse an expression, optionally followed by ASC or DESC (used in ORDER BY)
+    /// Parse an OrderByExpr expression, optionally followed by ASC or DESC (used in ORDER BY)
```

Comment:
```suggestion
    /// Parse an [OrderByExpr] expression.
```
Since the ASC/DESC is part of the returned `OrderByExpr`